### ISSUE TITLE
[question]

### ISSUE THREAD

**Issue by IvanIsak2000** [MazenYasser/steam-clone#1]
Hello! 
Have you used `Figma` in your project to create a design?
I ask this because in `figma` I can upgrade your design if you want this, of course.

**Comment by MazenYasser** [comment]
No, I have not used UI design software,I only looked up steam and recreated its design using HTML, CSS, Bootstrap 4 and jQuery.
You're welcome to do that.

**Comment by IvanIsak2000** [comment]
Hello bro!
I have a question about `steam-django/steam_clone_django/datadump.json`. 
It seems to me that the file should not be in the repository, can I delete it in next PR?

**Comment by MazenYasser** [comment]
Yes, this was an old backup of the database while migrating to postgreSQL.
